packages/core/getBusyTimes.ts

```diff
@@ -155,29 +155,31 @@ export async function getBusyTimes(params: {
       select: bookingsSelect,
     });
 
-    const currentBookingsAllUsersQueryThree = prisma.booking.findMany({
-      where: {
-        startTime: { lte: endTimeDate },
-        endTime: { gte: startTimeDate },
-        eventType: {
-          id: eventTypeId,
```

Comment:
If id is undefined, it can target all event-types. 

Something similar to https://github.com/calcom/cal.com/pull/15721 could help in this case as well.